qupath-gui-fx/src/main/java/qupath/lib/gui/actions/ActionTools.java

```diff
@@ -454,12 +451,27 @@ private static void parseConfig(Action action, ActionConfig annotation, Localize
 				localizedResourceManager.registerProperty(action.textProperty(), bundle, key);
 			else
 				action.setText(QuPathResources.getString(bundle, key));
+
 			String descriptionKey = key + ".description";
 			if (QuPathResources.hasString(bundle, descriptionKey)) {
-				if (annotation.bindLocale())
-					localizedResourceManager.registerProperty(action.longTextProperty(), bundle, descriptionKey);
-				else
-					action.setLongText(QuPathResources.getString(bundle, descriptionKey));
+				if (annotation.bindLocale()) {
+					StringProperty resourceLongTextProperty = new SimpleStringProperty();
+					resourceLongTextProperty.addListener((p, o, n) -> {
+						String longText = n;
+						if (action.getAccelerator() != null) {
+							longText = "(" + action.getAccelerator() + ") " + longText;
```

Comment:
Should we use the `.toString()` value or the `.getDisplayText()`?

See the difference with the following Groovy script:
```groovy
def combo = javafx.scene.input.KeyCombination.keyCombination("shortcut+c")
println combo
println combo.getDisplayText()
```
which prints
```
Shortcut+C
⌘C
```
On Windows this may be different, with the display text `Ctrl+C` but I haven't checked.

Reply:
`.getDisplayText()` seems better to me